iphone/Classes/TiUIiPhoneProxy.m

```diff
@@ -226,10 +226,12 @@ -(NSString*)apiName
 
 BEGIN_UI_THREAD_PROTECTED_VALUE(statusBarHidden,NSNumber)
 result = [NSNumber numberWithBool:[[UIApplication sharedApplication] isStatusBarHidden]];
+DEPRECATED_REPLACED_REMOVED(@"UI.iPhone.statusBarHidden",@"3.1.3", @"6.0.0", @"UI.Window.fullscreen");
```

Comment:
It should be on the `UI.Window` for longer as 6.0.0. Please consider, if this is necessary.